### PR TITLE
Clarify that there should normally be only one ActorSystem

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #24867

### CODE REVIEW

**Review by ktoso — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/typed/actors.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ Java
 We start an Actor system from the defined `main` behavior and send two `Start` messages that
 will kick-off the interaction between two separate `bot` actors and the single `greeter` actor.
 
+A normal application consists of a single `ActorSystem` which in turn contains many actors.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/actors.md]: a single ActorSystem per JVM?

I'm thinking about a clustered application, which consists of many JVMs

**Review by patriknw — APPROVED** [review]
LGTM with one possible nitpick

File: akka-docs/src/main/paradox/typed/actors.md (hunk 1 above)
Comment (johanandren) [reply akka-docs/src/main/paradox/typed/actors.md]: Good point, I'll change that

**Review by raboof — APPROVED** [review]

**Pull request merged**